conf.d/metrics.conf

```diff
@@ -9,7 +9,7 @@ command=twistd --nodaemon
     --password %(ENV_AMQP_PASSWORD)s
     --set-option=metrics_host:localhost
     --set-option=metrics_port:9109
-    --set-option="format_string:%%(metric_name)s %%(value)s %%(timestamp)s"
+    --set-option="format_string:(%%(metric_name)s %%(value)s %%(timestamp)s )\n"
```

Comment:
This doesn't seem correct to me. What we want is a line that looks something like:
```
test_udp 1234 1595318937
```
where if I'm understanding this correctly, we'll end up with a line that looks something like
```
(test_udp 1234 1595318937 )
```

I'm not sure why the `()` characters are added here?

Reply:
I added the parenthesis because simply adding a new line was not working (the metrics did not show up) But with adding the `()` and `echo` showed up like this ```test_udp 1234```

Reply:
do you feel this will be a one time occurance, and in real life it will fail and show up as ```(test_udp 1234 1595318937 )```?

Reply:
Yeah I'm not sure, that is what I would expect, but maybe there's something going on that I'm not understanding.

If you've tested this, and the netcat listen shows it without the `()`, then I'm happy with this